assembl/static2/js/app/pages/administration.jsx

```diff
@@ -168,6 +176,16 @@ class Administration extends React.Component {
     this.props.updateVoteSessionPage(voteSessionForStore);
   }
 
+  putTimelinePhasesInStore(timeline) {
+    const filteredPhases = filter(TimelineQuery, { timeline: timeline });
+    const phasesForStore = filteredPhases.timeline.map(phase => ({
+      ...phase,
+      start: phase.start ? moment(phase.start) : moment(),
```

Comment:
does it mean that sometimes you have `null` or `undefined` for start/end?

Reply:
No you are right, at this point it can't be null